ci/builders/linux_unopt.json

```diff
@@ -81,6 +81,50 @@
                 }
             ]
         },
+        {
+            "archives": [],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux",
+                "cores=32"
+            ],
+            "gclient_variables": {
+                "download_android_deps": false,
+                "use_rbe": true
+            },
+            "gn": [
+                "--runtime-mode",
+                "debug",
+                "--unoptimized",
+                "--prebuilt-dart-sdk",
+                "--target-dir",
+                "host_debug_impeller_vulkan",
+                "--rbe",
+                "--no-goma"
+            ],
+            "name": "host_debug_impeller_vulkan",
```

Comment:
Maybe rename this to something like `host_debug_impeller_tests`?

The build configuration and test suite are not doing anything specifically related to Vulkan.

Reply:
Good suggestion. Done.